dali/operators/segmentation/biased_crop_center.cc

```diff
@@ -0,0 +1,180 @@
+// Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <random>
+#include <utility>
+#include "dali/core/static_switch.h"
+#include "dali/pipeline/operator/operator.h"
+#include "dali/operators/segmentation/utils/searchable_rle_mask.h"
+#include "dali/kernels/common/utils.h"
+#include "dali/core/boundary.h"
+
+#define MASK_SUPPORTED_TYPES (uint8_t, int8_t, uint16_t, int16_t, uint32_t, int32_t, \
+                              uint64_t, int64_t, float)
+
+namespace dali {
+
+DALI_SCHEMA(segmentation__BiasedCropCenter)
+    .DocStr(R"(Selects a cropping window center which can be selected randomly from either
+any position in the input or any position of a foreground pixel in the input, based on an
```

Comment:
I don't know if we should use ``foreground`` or just non zero value, because the meaning of the ``foreground`` if not defined.